README.md

```diff
@@ -18,50 +18,82 @@
 ## Requirements
 
 * Mac OS X or Linux
-* Java 11, 64-bit
+* Java 11.0.8 or later, 64-bit
 * Python 2.6+ (for running with the launcher script)
-* Docker (for building and running tests)
+* Docker (for building and running tests, and for building documentation)
 
 ## Building Presto
 
-Presto is a standard Maven project. Simply run the following command from the project root directory:
+Presto is a standard Maven project. Simply run the following command from the
+project root directory:
 
     ./mvnw clean install
 
-On the first build, Maven will download all the dependencies from the internet and cache them in the local repository (`~/.m2/repository`), which can take a considerable amount of time. Subsequent builds will be faster.
+On the first build, Maven will download all the dependencies from the internet
+and cache them in the local repository (`~/.m2/repository`), which can take a
+considerable amount of time. Subsequent builds will be faster.
 
-Presto has a comprehensive set of unit tests that can take several minutes to run. You can disable the tests when building:
+Presto has a comprehensive set of unit tests that can also take a considerable
+amount of time to run. You can disable the tests when building:
 
     ./mvnw clean install -DskipTests
 
+If you experience a build fail, in many cases you can resume the build once you
+have corrected the build issue by simply leaving the 'clean' argument out of the
+build command:
+
+    ./mvnw install -DskipTests
+
+If your organization does not allow the use of Docker, you can skip building
```

Comment:
Maybe not talk about org .. just talk about .. if you do not have Docker installer...

Reply:
use